### PR TITLE
Zigbee: Update list of Known Issues

### PR DESCRIPTION
**Pull request by jangalda-nsc**

### CODE REVIEW

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -195,6 +195,32 @@ KRKNWK-6071: ZBOSS alarms inaccurate
...
 
+.. rst-class:: v1-4-0
+
+KRKNWK-7723: OTA upgrade process is not continued after OTA Upgrade Client reset
+  OTA Client will start OTA upgrade process from beginning after reset, instead continuing previous process.
+
+.. rst-class:: v1-4-0
+
+KRKNWK-7831: Factory reset not work on Coordinator with Zigbee Shell
+  Coordinator with Zigbee Shell component enabled could assert after executing 'bdb factory_reset' command.
```
Comment (sebastiandraus) [inline doc/nrf/known_issues.rst:206]: We may have workaround for that issue.

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -195,6 +195,32 @@ KRKNWK-6071: ZBOSS alarms inaccurate
...
+
+.. rst-class:: v1-4-0
+
+KRKNWK-7836: Coordinator with Zigbee Shell component enabled could assert when flooded with high number of commands.
```
Comment (sebastiandraus) [inline doc/nrf/known_issues.rst:210]: Should we narrow it down just to the ZDO commands?

**Review by sebastiandraus — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Comment by jangalda-nsc** [comment]
@carlescufi could you please merge it?

**Comment by jangalda-nsc** [comment]
Ok, CI finally passed. @carlescufi could you please merge it

**Pull request merged**